### PR TITLE
bump terpnetwork resources

### PR DESCRIPTION
**Pull request by discoverdefiteam**

### CODE REVIEW

**Comment by hard-nett** [comment]
These commits will not be in effect for main-net moroccco-1 until after our planned upgrade this weekend

File: terpnetwork/chain.json [hunk 1]
```diff
@@ -103,18 +103,25 @@
           "type": "cometbft",
           "version": "0.37.2"
         }
+      },
+      {
+        "name": "v4.1.0",
```
Comment (reversesigh) [inline terpnetwork/chain.json:108]: Can you add an entry for "next_version_name": to the previous version, indicating the next version as "v4.1.0"? And can you add two new entries to v4.1.0 for "proposal" and "height" to indicate the upgrade prop # and upgrade height? Also feel free to add a dummy blank "next_version_name" to v4.1.0. Refer to https://github.com/cosmos/chain-registry/blob/master/ununifi/chain.json as a good example.

File: terpnetwork/chain.json [hunk 2]
```diff
@@ -102,7 +102,10 @@
         "consensus": {
           "type": "cometbft",
           "version": "0.37.2"
-        }
+        },
+        "next_version_name": "v4.1.0",
+        "proposal": 30,
+        "height": 3698609
```
Comment (reversesigh) [inline terpnetwork/chain.json:108]: 107 and 108 (prop and height) should be moved to the next section for v4.1.0. Then we can merge.

**Comment by hard-nett** [comment]
@reversesigh hi! can this be merged? or should we create fresh PR? Thank you! :)

**Pull request merged**